src/FixtureBuilder/Denormalizer/Fixture/SpecificationBagDenormalizer/SimpleSpecificationsDenormalizer.php

```diff
@@ -64,6 +66,24 @@ public function denormalize(FixtureInterface $scope, FlagParserInterface $parser
             if ('__construct' === $unparsedPropertyName) {
                 $constructor = $this->denormalizeConstructor($value, $scope, $parser);
 
+                if (false === ($constructor instanceof NoMethodCall) && '__construct' !== $constructor->getMethod()) {
```

Comment:
I kinda prefer being explicit instead of using `!`, as it's the current convention I would prefer not to change it at that point :)